### PR TITLE
Add various small changes for orcharhino

### PR DESCRIPTION
**Pull request by maximiliankolb**
Please review commit by commit.

* [x] I am familiar with the [contributing](https://github.com/theforeman/foreman-documentation/blob/master/CONTRIBUTING.md) guidelines.

Please cherry-pick my commits into:

* [x] Foreman 3.6/Katello 4.8
* [x] Foreman 3.5/Katello 4.7 (planned Satellite 6.13)
* [x] Foreman 3.4/Katello 4.6 (EL8 only)
* [x] Foreman 3.3/Katello 4.5 on EL7 & EL8 (Satellite 6.12 on EL8 only)

### CODE REVIEW

File: guides/common/modules/snip_host-registration-steps.adoc [hunk 1]
```diff
@@ -10,11 +10,13 @@ endif::[]
...
+. Ensure that the appropriate repositories have been enabled:
++
 ifdef::client-content-dnf[]
-. Check the `/etc/yum.repos.d/redhat.repo` file and ensure that the appropriate repositories have been enabled.
+** On {EL}: Check the `/etc/yum.repos.d/redhat.repo` file and ensure that the appropriate repositories have been enabled.
 endif::[]
 ifdef::client-content-apt[]
-. Check the `/etc/apt/sources.list` file and ensure that the appropriate repositories have been enabled.
+** On Debian: Check the `/etc/apt/sources.list` file and ensure that the appropriate repositories have been enabled.
 endif::[]
```
Comment (adamlazik1) [inline guides/common/modules/snip_host-registration-steps.adoc:20]: The format does not seem that great if only one client content type is defined. Can there be separate ifdefs for deb, rpm, and katello instead?

Comment (maximiliankolb) [reply guides/common/modules/snip_host-registration-steps.adoc]: I've added more logic to accomodate this. But in downstream, I basically have the same issue: There is only _one_ content type active per render, be it dnf, apt, or zypper.

Maybe I can create a more elegant logic than this in the future!

**Review by maximiliankolb — COMMENTED** [review]
Rebased to master and applied all feedback.

File: guides/common/modules/snip_host-registration-steps.adoc [hunk 2]
```diff
@@ -42,11 +49,18 @@ You can use the ID of any {SmartProxyServer} that you configured for host regist
 Include the `--force` option to register a host that has been previously registered to a {SmartProxyServer}.
 endif::[]
 . Connect to your host using SSH and run the registration command.
-ifdef::client-content-dnf[]
+ifdef::satellite[]
 . Check the `/etc/yum.repos.d/redhat.repo` file and ensure that the appropriate repositories have been enabled.
 endif::[]
+ifndef::satellite[]
```
Comment (adamlazik1) [inline guides/common/modules/snip_host-registration-steps.adoc:55]: Works for me! Feel free to create a section for orcharhino if you also want it displayed in a similar fashion.

**Review by adamlazik1 — APPROVED** [review]
LGTM

**Comment by maximiliankolb** [comment]
Merged to master and cherry-picked: :heavy_check_mark: 
   432702b23..8b5755f2e  3.6 -> 3.6
   0d91dbc03..af26ff84e  3.5 -> 3.5
   0e8e94d1a..8fbc93a71  3.4 -> 3.4
   5b643ebe0..886bd7973  3.3 -> 3.3

**Pull request merged**